models/raw/segur/forca_municipal/README.md

```
@@ -0,0 +1,219 @@
+
+Modelos de materialização das tabelas de dump da pipeline `rj_segur__forca_municipal`.
+
+**Fonte:** API HxGN OnCall (CIVITAS/CORIO) — sistema de gestão de ocorrências e unidades da Guarda Municipal do Rio de Janeiro.
+**Staging:** `rj-iplanrio.brutos_forca_municipal_staging`
+**Destino:** `rj-iplanrio.brutos_forca_municipal`
+
+---
+
+## Convenções de nomeação de arquivos
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)